### DISCUSSION TITLE
Damages file with px6s (Mono X 6Ks)

### DISCUSSION THREAD

**Discussion by KAW41I** [sn4k3/UVtools#782] [Q&A] [answered]
i have bought the Anycubic Photon Mono x 6Ks, when i run the files through uvtools, it says by the printing, file is damaged! printed stopped, also the uvtools file is a lot whole shorter, 416 and 417 version from anycubic in uvtools
i Have Version 4.0.3 windows x64 of UVTools

The normal LvlTest.px6s from chitubox is printing.
chitubox project + setting also in files
Files: https://drive.google.com/drive/folders/1zGjM-U_rmyxZyM4CCKlMlt3iyRjNR3mj?usp=sharing

so i tried the photon slicer from anycubic, same orginal file works, though the uvtools and then the file is damaged

**Comment by HWKInuria** [comment]
A workaround you can try in the meantime is reloading the sliced UVTools px6s file in Chitubox and save it again, has worked for me with previous Versions if my printer wasnt supported in UVTools yet.

**Reply by KAW41I** [reply]
Thanks, worked but it needs longer to load the file again in chitubox, i mean it works for the meantime, but over 20 Minutes wait time

**Comment by sn4k3** [comment, marked as answer]
The cause why UVtools file is smaller is because your layers are all equal / same image across the 20 layers. On save UVtools cache images and for equal images it saves only one image and point equal layers to that image, because each layer has an image pointer and then it's possible to save bytes. If firmwares are codded correctly it will jump to image address instead of sequential fetch, but from what I saw this past year's firmwares are never to trust as often they don't fully respect own file spec.
I provide you two files, print both without VAT and report back which file printed. One is saved with cache and other without the cache.

[px6s-files.zip](https://github.com/sn4k3/UVtools/files/13271060/px6s-files.zip)

**Reply by KAW41I** [reply]
the _uncached startet printing, the _cached said file damaged
I only let the first layer light up and then canceld the print FYI

**Reply by sn4k3** [reply]
Ok so that's the problem, firmware is checking or not prepared to jump to image address.
Next patch I will remove the cache feature from this format

**Reply by sn4k3** [reply]
Also try this last:

[LvlTestPhotonWorkshop_00h10m_5ml_nullcached.zip](https://github.com/sn4k3/UVtools/files/13272234/LvlTestPhotonWorkshop_00h10m_5ml_nullcached.zip)

**Reply by KAW41I** [reply]
_nullcached dint work, file is damaged

**Reply by sn4k3** [reply]
Ok, what about this:

[LvlTestPhotonWorkshop_00h10m_5ml_lastcached.zip](https://github.com/sn4k3/UVtools/files/13272947/LvlTestPhotonWorkshop_00h10m_5ml_lastcached.zip)

**Reply by KAW41I** [reply]
_lastcached didnt work, file is damaged

only one that worked so far was  _uncached

**Reply by sn4k3** [reply]
Try the new release

**Reply by efogdev** [reply]
Worked for me, thanks a lot

**Reply by KAW41I** [reply]
yea thanks worked

**Comment by MakManIfest** [comment]
Not related to uv tools per say, but I also get damaged file errors, seemingly at random when I slice with Lychee for my 6ks.

Could the image caching be the same issue with lychee. I contacted lychee support but have not been able to resolve the issue.

**Reply by sn4k3** [reply]
AFIK lychee does not do cache of the slices. However they got some values wrong on the table that I reported to them, most likely would not cause this either (I think). You should report to them with the file for analisis.

**Reply by MakManIfest** [reply]
I have provided them with multiple files that have the error. They suggested updating my video card drivers, which I did. But this Did not solve the issue. I'll see if there is a firmware update for the printer. Haven't check for a couple of weeks.

**Reply by sn4k3** [reply]
If error is random is hard to detect, but inspecting the file can have a clue. If you want attach file here
